docs/requirements.txt

```diff
@@ -0,0 +1,6 @@
+sphinx>=3.2.0
+pytorch-sphinx-theme>=0.0.19
```

Comment:
`pytorch_sphinx_theme @ git+https://github.com/pytorch/pytorch_sphinx_theme.git#egg=pytorch_sphinx_theme`

I believe this is not regularly released to PyPI.